test/plugins/prometheus/index.test.ts

```diff
@@ -42,7 +43,7 @@ describe('Prometheus plugin', () => {
       )
       .it('runs Prometheus metric server', async (ctx) => {
         // act
-        const res = await fetch('http://localhost:4444/metrics')
+        const res = await axios.get('http://localhost:4444/metrics')
```

Comment:
Don't change this to Axios. Because, there is an effort to change from Axios to fetch in this PR https://github.com/hyperjumptech/monika/pull/1094. Could you elaborate why you need to change this?

Reply:
I think #1094 is to change the implementation when probing. But this axios is used for test. it's not related, right?

Reply:
this is the root cause of https://github.com/hyperjumptech/monika/pull/1110 
maybe need to merge it first?

Reply:
Ok